src/main.rs

```diff
@@ -531,7 +547,7 @@ mod tests {
         let mut actual_file = BufWriter::new(actual_file.unwrap());
         _ = write!(actual_file, "{}", actual_string);
 
-        assert_eq!(expected_json_value, actual_json_value);
+        assert_eq!(expected_json_value, actual_json_value); // Testing JSON
```

Comment:
I don't think these comments add any information.

Reply:
Yes I will update the whole PR with an equivalent logic to your tidyTest